File: BankAccountController.py
from BankAccount import BankAccount
from Employee import Employee
from Bank import Bank

class BankAccountController:
    @staticmethod
    def create_account(nC, type):
        cuenta = BankAccount(nC, type)

        print("Elige el empleado para darle la cuenta:")

        for id, emple in enumerate(Bank.get_employee_list()):
            print(f"ID {id + 1}: {emple.get_name()} {emple.get_lastname()}\n")

        emID = int(input()) - 1
        emp = Bank.get_employee_list()[emID]
        emp.add_account(cuenta)
        Bank.get_bank_accounts().append(cuenta)

        print(f">>> Se asignado la cuenta {nC} a {emp.get_name()} {emp.get_lastname()}")

    @staticmethod
    def check_all_accounts():
        for accounts in Bank.get_bank_accounts():
            print(f"{accounts.get_account_number()}, {accounts.get_type()}, {accounts.get_amount()}")

    @staticmethod
    def modify_account():
        print("¿Qué cuenta quiere modificar?")
        for id, cuentas in enumerate(Bank.get_bank_accounts()):
            print(f"§ ID {id + 1} | No. cuenta: {cuentas.get_account_number()} §")
        account = int(input()) - 1
        modify_bank_account = Bank.get_bank_accounts()[account]

        print("Nuevo numero de cuenta:", end=" ")
        modify_bank_account.set_account_number(int(input()))

        print("Nuevo tipo de cuenta:", end=" ")
        modify_bank_account.set_type(input()[0])

        print(">>> Cuenta exitosamente cambiada")

File: BankAccountController.java
public class BankAccountController {


    public  void crearCuenta(long nC,char type) {


        BankAccount Cuenta = new BankAccount(nC, type);

        System.out.println("Elige el empleado para darle la cuenta:");

        int ID = 0;
        for (Employee emple : Bank.getEmployeeList()) {

            System.out.println("ID " + (ID+1) + ": " + emple.getName() + " " + emple.getLastname() + "\n");
            ID++;
        }
        int emID = ConsoleReader.sc.nextInt();
        emID--;

        Employee emp = Bank.getEmployeeList().get(emID);
        emp.addAccount(Cuenta);
        Bank.getBankAccounts().add(Cuenta);

        System.out.println(">>> Se asignado la cuenta "+nC+" a "+emp.getName()+" "+emp.getLastname());
    }

    public  void checkAllAccounts() {

        for (BankAccount accounts : Bank.getBankAccounts()) {//for(Tipo_Objeto nombre_Objeto : Nombre_ArrayList){}
            System.out.println(accounts.getAccountNumber() + ", " + accounts.getType() + ", " + accounts.getAmount());
        }

    }

    public  void modifyAccount(){

        System.out.println("Que cuenta quiere modificar?");
        int ID=0;
        for(BankAccount cuentas : Bank.getBankAccounts()){
            System.out.printf("§ ID %s | No. cuenta: %s §\n",(ID+1),cuentas.getAccountNumber());
            ID++;
        }
        int account = ConsoleReader.sc.nextInt();
            account--;
        BankAccount modifyBankAccount = Bank.getBankAccounts().get(account);

        System.out.print("Nuevo numero de cuenta: ");
        modifyBankAccount.setAccountNumber(ConsoleReader.sc.nextLong());

        System.out.print("Nuevo tipo de cuenta: ");
        modifyBankAccount.setType(ConsoleReader.sc.next().charAt(0));

        System.out.println(">>> Cuenta exitosamente cambiada");
    }
}
